deps/undici/src/package.json

```diff
@@ -67,11 +67,10 @@
     "index-fetch.js",
     "lib",
     "types",
-    "docs",
-    "scripts/esbuild-build.mjs"
+    "docs"
   ],
   "scripts": {
-    "build:node": "node scripts/esbuild-build.mjs",
+    "build:node": "npx esbuild@0.19.4 index-fetch.js --bundle --platform=node --outfile=undici-fetch.js --define:esbuildDetection=1 --keep-names",
```

Comment:
How is this build dependency fixed to be esbuild@0.19.4 version automated to be maintained up to date?

Reply:
This was done like that in undici, and we switched to a build script in a hurry for the security release. https://github.com/nodejs/undici/blob/4006aaf43ac8b30e16d6d3b89fa2e0df4b7eef33/package.json#L73.

Reply:
@panva undici always pinned esbuild's version here. I updated it from some version of 0.14. Realistically we would want to get rid of esbuild entirely, as it's caused nothing but issues (hinting at my next pr probably).